code/src/index.css

```diff
@@ -1,13 +1,175 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 64px 24px 24px 24px;
+}
+
+body, button, input {
+  font-family: 'Space Mono', monospace;
+  font-size: 24px;
+  line-height: 150%;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: #2E233D;
+  color: #FFECDE;
+}
+
+p, h1, h2, h3, h4 {
+  margin: 0;
+}
+
+.gameover-text {
+  font-size: 24px;
+  text-align: center;
+  margin-top: 40px;
+  margin-bottom: 40px;
+  line-height: 150%;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+main {
+  width: 100%;
+  max-width: 988px;
+  margin: 0 auto;
 }
+
+.game-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.box-description, .box-action, .box-username {
+  background: #2E233D;
+  padding: 24px;
+  display: flex;
+  border: 2px solid #FFECDE;
+}
+
+.box-username {
+  flex-direction: column;
+}
+
+.box-username > * {
+  margin: 30px 0 0 0;
+}
+
+.box-username > *:first-child {
+  margin: 0;
```

Comment:
This looks like a fancy way to do some css styling! You have to tell us what this is!?

Reply:
Great job!